assets/css/all.sass

```diff
@@ -59,19 +69,26 @@ a
 a
   &:hover
     color: $secondary-color
+a
+  &:visited
+    color: var(--link-visited-color)
```

Comment:
By inverting these two, ISTM we get the hover action on the visited links just like on the not-yet visited ones.
```diff
-a
-  &:hover
-    color: $secondary-color
 a
   &:visited
     color: var(--link-visited-color)
 
+a
+  &:hover
+    color: $secondary-color
+
```

Reply:
Great idea, added in 1b7d6d520ac056a947a66135aa3bfd979ecdcdce

tbh, I didn't even realize the `.sass` files were top-down hierarchical 🤦